1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,26 @@
+let quoteEl=document.getElementById("quote_text");
+let nameEl=document.getElementById("writer_name");
+let button=document.getElementById("new_quote");
+let bodyEl=document.getElementById("content")
+button.addEventListener("click", () => {
```

Comment:
Nicely done. Do You have auto-formatting on?

Reply:
yes, before submitting my work, I am formatting my files. thank you, Patryk